src/platforms/java/common/maven/index.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Maven
+description: "Learn about using the Sentry Maven Plugin."
+sidebar_order: 105
+---
+
+The [Sentry Maven Plugin](https://github.com/getsentry/sentry-maven-plugin) is an addition to the main Java SDK and offers
+seamless integration with the Maven build system. It supports the following features:
+
+- Auto-installation of Sentry Java SDK and relevant integrations
+- Uploading Source Context
+
+## Setup
+
+### Install
+
+Using Maven in your application module's `pom.xml` add:
+
+```xml
+<project>
+  ...
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.sentry</groupId>
+        <artifactId>sentry-maven-plugin</artifactId>
+        <version>{{@inject packages.version('sentry.java.maven-plugin', '0.0.2') }}</version>
+        <extensions>true</extensions>
+      </plugin>
+      ...
+    </plugins>
+    ...
+  </build>
+...
+</project>
+```
+
+### Configure
+
+We expose the following configuration values for the plugin in `pom.xml`:
```

Comment:
`l` Maybe explain here that this sample adds a `<configuration>` element to the `pom.xml` sample above?